src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.groovy

```diff
@@ -3,7 +3,7 @@ package com.atlassian.bitbucket.jenkins.internal.config.BitbucketTokenCredential
 def f = namespace(lib.FormTagLib)
 
 f.entry(title: _("bitbucket.admin.token"), field: "secret") {
-    f.password(value: instance?.encryptedPassword)
+    f.password(value: instance?.secret)
```

Comment:
What's this change for?

Reply:
We renamed the field 'secret' in [BitbucketTokenCredentialsImpl](https://github.com/jenkinsci/atlassian-bitbucket-server-integration-plugin/blob/4fafff7af76eaee24585aa6dc471227bc423c988/src/main/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl.java#L21).
On a new instance this was not an issue (the `instance?` checks if there is an instance or not) which seems to be all we tried. When you tried to edit a token you'd get an exception as there was no field named `encryptedPassword`. This change lines up the field name with the name in the groovy.